content/posts/survey-01/index.md

```diff
@@ -0,0 +1,267 @@
+
+Priorities for professionals and hobbyists are largely the same. The biggest differences were:
+
+- Hobbyists call for ecosystem maturity 2x more than anything else.
+
+- Hobbyists care much more about mobile/web support
+
+- Professionals care much more about console support
+
+- Professionals care much more about C++ interop
+
+
+### Ecosystem maturity (114)
+
+* From playing in the space I know it's possible to make games using Rust. However in the ecosystem my biggest negatives is the [lack of] case studies and demonstrations of it working.
+
+* Rust is still a risky choice, compared to C++.
+
+* Still feels like a new community. There is always risk associated with that - does the language take a big swing in direction - does support drop off. I'm encouraged by the support for low level (embedded / OS ) development, which hopefully means the core feature set remains robust.
+
+* Many unfinished, unmaintained crates. Lots of churn.
+
+* A lot of common needs are not yet available 'out of the box'.
+
+### Iteration times (68)
+
+* Compile time, no runtime reflection, updating deep dependency in the tree is hard.
+
+* Iteration time is I think too big, due to compilation time, [lack of] hot-reloading tooling, etc.
+
+* Iteration time (especially while prototyping). This includes both compile time but also sorting out lifetimes when I know I'll be rewriting something very soon anyway, this "frontloading" the effort of having the borrow checker run at compile time is useful for most tasks but when I want to try something quickly it can be a draining experience, in this regard GC'd languages have an advantage imo.
+
+* Prototyping seems not as easy in Rust due to "fighting the compiler"
+
+###  Documentation (43)
+
+* Not enough documentation to get past beginner stage
+
+* Lacking examples for gamedev libs.
+
+* Documentation is usually either high level or examples but rarely both. Very few real world examples to draw from.
+
+### Lack of full-featured engines (35)
+
+* Lack of a data-and-script based engine like Unreal Engine or Unity means even small projects end up feeling very bring-your-own-engine.
+
+* Missing graphical editors, frictionless prototyping of games
+
+
+### Rust learning curve / onboarding to a niche language (29)
+
+* The language has a higher upfront cognitive load than any other mainstream language I know of, mostly due to the borrow checker. It's very hard to hack through stuff and get away with it, which makes it a particularly hard sell for game prototypes.
+
+* I know that it is a less common language to use so as a company I will have to have my staff learn Rust before they can start.
+
+* Rust could be tricky to hire for, and hiring someone who doesn't know Rust to work in Rust will come with a ramp-up period.
+
+### IDE/RLS story (28)
+
+Tooling around IDE is limited compared to Visual Studio with C++ or C#.´
```

Comment:
There's a bullet point missing here and on lines 112/114.

I also kinda wonder if there's a way to format these bullets to make it a little clearer that they're quotes, but it's probably fine as is.

Reply:
I think marking it as quotes is actually a good point. The first time I read it, I wasn't sure whether they were a distilled, bullet-point summary of reflective quotes.

Reply:
With the site's current styling, quotes would look like this:

![image](https://user-images.githubusercontent.com/784533/73682565-835c1c00-46b8-11ea-98ec-6f4a6c85e918.png)

If you do decide to go that route, please let me know if you're not happy with the fonts/italics/etc - I can submit a quick PR to change them before this post gets merged :)